webapp/src/app/components/toolbar/toolbar.component.ts

```diff
@@ -72,6 +73,10 @@ export class ToolbarComponent implements OnInit {
 		}
 
 		this.events.babylonLoading.subscribe(val => this.is3dLoading = val);
+
+		this.events.updateCoords.subscribe(coords => {
+			this.coords = coords === null ? '' : `(${coords.x}, ${coords.y}, ${coords.z})`;
```

Comment:
This could also be rewritten to 

```suggestion
			this.coords = coords ? `(${coords.x}, ${coords.y}, ${coords.z})` : '';
```

which makes it a bit easier to read in my opinion

Reply:
I've done `!coords ? '' : '...'` but its basically the same thing